### PR TITLE
Issue templates

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#522 

### Added
- `feature_request.yaml` template
- `config.yml` for issue template
### Changed
- bug report template modified

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      default: 1
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of PyCM are you using?
+      options:
+        - 4.0 (Default)
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:84]: Is it necessary to write that `(Default)`?

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      default: 0
+    validations:
+      required: true
+  - type: dropdown
+    id: python-version
+    attributes:
+      label: Python version
+      description: Which version of Python are you using?
+      options:
+        - 3.12.0
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:67]: I suggest `3.x` instead of `3.x.x`

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    validations:
+      required: true
+  - type: dropdown
+    id: python-version
+    attributes:
+      label: Python version
+      description: Which version of Python are you using?
+      options:
+        - 3.12.0 
+        - 3.11.0 (Default)
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:68]: Is it necessary to write that `(Default)`?

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (sepandhaghighi) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:84]: I believe that we should include a pattern in the `version_check.py` script.

Comment (sepandhaghighi) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:84]: Please take a look at https://github.com/sepandhaghighi/pycm/releases.
We have 43 releases!

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: step-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Provide details of how to reproduce the bug.
+      placeholder: ex. 1. Go to '...'
+    validations:
+      required: false
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:32]: I believe this field is required

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: What you expected to happen?
+      placeholder: ex. I expected '...' to happen
+    validations:
+      required: false
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:40]: I believe this field is required

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      description: What actually happened?
+      placeholder: ex. Instead '...' happened
+    validations:
+      required: false
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:48]: I believe this field is required

File: CHANGELOG.md [hunk 7]
```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `feature_request.yaml` template
+- `config.yml` for issue template
 ### Changed
+- bug report template modified
```
Comment (sepandhaghighi) [inline CHANGELOG.md:12]: - `Bug` instead of `bug`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your efforts 💯 
Please take a look at my comments 🔥

**Comment by sadrasabouri** [comment]
Changes applied @sepandhaghighi 
As you can see in the sandbox repository (for testing issue templates) the required field is not working.
I will work more on it to see what should I do and update this PR.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+        - Python 3.7
+        - Python 3.6
+        - Python 3.5
+      default: 1
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:81]: I believe that `PyCM version` is superior.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 9]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord
+    url: https://discord.com/invite/zqpU2b3J3f
+    about: Ask questions and discuss with other pycm community members 
+  - name: Mailing List
+    url: https://mail.python.org/mailman3/lists/pycm.python.org/
+    about: Ask questions and discuss with other pycm community members
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/config.yml:8]: PyCM

File: .github/ISSUE_TEMPLATE/config.yml [hunk 10]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord
+    url: https://discord.com/invite/zqpU2b3J3f
+    about: Ask questions and discuss with other pycm community members
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/config.yml:5]: PyCM

File: Otherfiles/RELEASE.md
Comment (sepandhaghighi) [inline Otherfiles/RELEASE.md:1]: Please update `#### Last Update:` section

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your efforts 🔥 
Please take a look at my comments 💯 
Is it possible to unify all template file formats to either `.yaml` or .`yml` formats? Please update all files accordingly.

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🚀 
@alirezazolanvari Please review this PR 🔥

**Comment by sepandhaghighi** [comment]
@alirezazolanvari Just a friendly reminder ⏲️

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 11]
```diff
@@ -0,0 +1,135 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
```
Comment (alirezazolanvari) [inline .github/ISSUE_TEMPLATE/bug_report.yml:8]: My Suggestion:

" Thanks for your time to fill out this bug report!"

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+      label: Steps to reproduce
+      description: Provide details of how to reproduce the bug.
+      placeholder: ex. 1. Go to '...'
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: What you expected to happen?
```
Comment (alirezazolanvari) [inline .github/ISSUE_TEMPLATE/bug_report.yml:37]: What did you expect to happen?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+      label: Actual behavior
+      description: What actually happened?
+      placeholder: ex. Instead '...' happened
+    validations:
+      required: true
+  - type: dropdown
+    id: operating-system
+    attributes:
+      label: Operating system
+      description: What operating system are you using?
```
Comment (alirezazolanvari) [inline .github/ISSUE_TEMPLATE/bug_report.yml:53]: Which operating system are you using?

**Review by alirezazolanvari — CHANGES_REQUESTED** [review]
I left some comment about minor issues. Please check them out

**Review by sepandhaghighi — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
@alirezazolanvari Just a friendly reminder ⏲️

**Pull request merged**